tests/test_auto_context.py

```diff
@@ -65,11 +65,14 @@ async def page(client: Client):
     screen.should_contain('before connected')
     screen.should_contain('after connected')
     screen.should_not_contain('one')
-    screen.wait_for('one')
+    screen.wait(0.5)
```

Comment:
Why do we need to wait in such cases? Shouldn't `should_see` wait automatically?

Reply:
Yes. the `screen.wait(0.5)` can be omitted. But then the code would read:

```py
    screen.should_not_contain('one')
    screen.should_contain('one')
```

Which is a little bit strange in my opinion. We are not wasting time, because the sleep in the NiceGUI code is 1s.